### PR TITLE
feat: add support for pallet-xcm claimAssets call

### PR DESCRIPTION
**Pull request by marshacb**
## Description

This PR adds support for constructing local `claimAssets` runtime calls to the `AssetTransferApi` for claiming trapped assets.

### Claimable Assets
Assets to claim can be provided as asset symbols (e.g. `usdt`, `wnd`), locations (e.g. `{"parents":"0","interior":{"X2":[{"PalletInstance":"50"},{"GeneralIndex":"66"}]}}`) or integers (e.g. `66`, `1984`).

Note: An asset can also be provided as an empty string. In this case the assetId will be recognized as the relay chains native asset. (e.g. `['']` will be equivalent to `wnd` when on a Westend chain).

Example of constructing a `claimAssets` call to claim trapped `WND` on `Westend`:
```typescript
api.claimAssets(
	['WND'],
	['1000000000000'],
	'5HBuLJz9LdkUNseUEL6DLeVkx2bqEi6pQr8Ea7fS4bzx7i7E', 
	{
		xcmVersion: 4,
		format: 'payload',
		sendersAddr: '5EWNeodpcQ6iYibJ3jmWVe85nsok1EDG8Kk3aFg8ZzpfY1qX'
	} 
)
```

Expected output: 
```json
{"origin":"0","dest":"westend","direction":"local","xcmVersion":4,"method":"claimAssets","format":"payload","tx":"0xc8630c0404000000070010a5d4e80400010100e2aa84cbfa479f0150bb77eb06756ee2c725ec625e16dd68b2678a0820e8d834b500940050690f0018000000e143f23803ac50e8f6f8e62695d1ce9e4e1d68aa36c1cd2cfd15340213f3423e901c8a5932603948278d24d86dfc72e508d890723bf6bd3cf9f8645be53cbdc2"}
```

closes: [#373](https://github.com/paritytech/asset-transfer-api/issues/373)

### CODE REVIEW

File: examples/claimAssets.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ * When importing from @substrate/asset-transfer-api it would look like the following
+ *
+ * import { AssetTransferApi, constructApiPromise } from '@substrate/asset-transfer-api'
+ */
+import { AssetTransferApi, constructApiPromise } from '../src';
+import { TxResult } from '../src/types';
+import { GREEN, PURPLE, RESET } from './colors';
+
+/**
+ * In this example we are creating a `claimAssets` call to claim `wnd` tokens trapped in the AssetTrap of Westend AssetHub where the `xcmVersion` is set to 3.
```
Comment (TarikGul) [inline examples/claimAssets.ts:11]:
```suggestion
 * In this example we are creating a `claimAssets` call to claim `WND` tokens trapped in the AssetTrap of Westend AssetHub where the `xcmVersion` is set to 3.
```

File: examples/claimAssets.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ *
+ */
+const main = async () => {
+	const { api, specName, safeXcmVersion } = await constructApiPromise('wss://westend-asset-hub-rpc.polkadot.io');
+	const assetApi = new AssetTransferApi(api, specName, safeXcmVersion);
+	let callInfo: TxResult<'call'>;
+	try {
+		callInfo = await assetApi.claimAssets(
+			['wnd'],
```
Comment (TarikGul) [inline examples/claimAssets.ts:20]:
```suggestion
			['WND'],
```

**Review by TarikGul — APPROVED** [review]
LGTM, great job. Just a a few small nits in the examples

**Comment by Polkadot-Forum** [comment]
This pull request has been mentioned on **Polkadot Forum**. There might be relevant details there:

https://forum.polkadot.network/t/paritytech-update-for-april/7646/1

**Pull request merged**